server/handler.go

```diff
@@ -236,19 +236,24 @@ func (h *Handler) AddScheduler(name string, args ...string) error {
 		return err
 	}
 	log.Info("create scheduler", zap.String("scheduler-name", s.GetName()), zap.Strings("scheduler-args", args))
-	if !h.s.IsAPIServiceMode() {
+	if h.s.IsAPIServiceMode() {
```

Comment:
Will we not need to forward create-scheduler and remove-scheduler requests to the scheduler server

Reply:
No, the meta info should all be managed in the API server, and the scheduling service will watch them.